components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/connector/recaptcha/PasswordRecoveryReCaptchaConnector.java

```diff
@@ -100,7 +101,8 @@ public boolean canHandle(ServletRequest servletRequest, ServletResponse servletR
 
         String path = ((HttpServletRequest) servletRequest).getRequestURI();
 
-        return !StringUtils.isBlank(path) &&
+        List<String> reCaptchaBypassedApiEndpoints = CaptchaDataHolder.getInstance().getReCaptchaBypassedApiEndpoints();
```

Comment:
Hope this list is not null and initialized as at least empty array.

Reply:
Yes, this is initialized as empty list.